lxd/instance/drivers/driver_lxc.go

```diff
@@ -2235,6 +2241,13 @@ func (d *lxc) startCommon() (string, []func() error, error) {
 		}
 	}
 
+	if len(cdiConfigFiles) > 0 {
+		err = lxcSetConfigItem(cc, "lxc.hook.mount", fmt.Sprintf("%s callhook %s %s %s startmountns --devicesRootFolder %s %s", d.state.OS.ExecPath, shared.VarPath(""), strconv.Quote(d.Project().Name), strconv.Quote(d.Name()), d.DevicesPath(), strings.Join(cdiConfigFiles, " ")))
+		if err != nil {
+			return "", nil, err
```

Comment:
feels like we should be wrapping the error with context here, instance, project, device etc.

Reply:
fixed